src/api/v1alpha1/inspectionpolicy_types.go

```diff
@@ -180,6 +163,7 @@ type Strategy struct {
 	// +kubebuilder:validation:Optional
 	HistoryLimit *int32 `json:"historyLimit"`
 	// Suspend the subsequent inspections temporarily.
+	// +kubebuilder:default:=false
```

Comment:
As we discussed, this is not the exact fix which I shared.
With this change, the strategy won't be populated by default and won't take any default values as I demonstrated during our meeting.

The idea is to populate the strategy with the default set of attributes.